### ISSUE TITLE
Create more callback templates which accepts a standardised set of params

### ISSUE THREAD

**Issue by d4nyll** [d4nyll/instafetch#5]
Currently, there is just one `example.js` with one example callback function. It would be nice if there are more templates.

Also, these templates should accept params with a defined structure so different templates can work 'plug-and-go' (modular) style.

**Comment by tamfr** [comment]
I agree, if you look at https://github.com/stevenschobert/instafeed.js#templating which is trying but failing to do what your script easily handles (grabbing images with a certain tag from a user and grab them from all past posts), it has a very convenient templating format.  I also am pretty sure that the way your script builds the elements and thus, the template, dynamically through the document.createElement and appendChild functions, slows it down.  The real speed increase will come through implementing a template system such as the one in instafeed. I'll see what if I can figure something out, but I'm not very familiar with coding js.

**Comment by d4nyll** [comment]
Very good suggestion, thank you for that Eric!
